src/launchpad/components/Samples.js

```diff
@@ -0,0 +1,78 @@
+let examples = {
```

Comment:
Minor nits 
- we should choose examples or samples
- maybe put this outside of components